### PR TITLE
Add `url` and `query_string` processors

### PR DESCRIPTION
**Pull request by jderusse**
Documentation for https://github.com/symfony/symfony/pull/28975

### CODE REVIEW

File: configuration/external_parameters.rst [hunk 1]
```diff
@@ -537,6 +537,129 @@ Symfony provides the following env var processors:
...
+
+    .. versionadded:: 4.3
+
+        The ``url`` processor was introduced in Symfony 4.3.
+
+``env(query_string:FOO)``
+    Parses an encoded string as if it were the query string passed via a URL.
+
+
+    .. code-block:: bash
```
Comment (OskarStark) [inline configuration/external_parameters.rst]: @javiereguiluz is bash ok or better terminal or text here for the code block?

File: configuration/external_parameters.rst [hunk 2]
```diff
@@ -537,6 +537,129 @@ Symfony provides the following env var processors:
...
+                'connections' => [
+                    'default' => [
+                        'database_name' => '%env(key:path:url:MONGODB_URL)%',
+                    ],
+                ],
+            ]);
+
+    .. caution::
+
+        In order to ease extraction of the resource from the URL, The leading
```
Comment (OskarStark) [inline configuration/external_parameters.rst]: The -> the

File: configuration/external_parameters.rst (hunk 1 above)
Comment (jderusse) [reply configuration/external_parameters.rst]: Note: this is the same code block used above in that file

Comment (OskarStark) [reply configuration/external_parameters.rst]: Thanks for the info, maybe we should change them all. Not sure

Comment (javiereguiluz) [reply configuration/external_parameters.rst]: For information purposes: we decided a few days ago to use `bash` for "bash things" (such as displaying the content of `.env` files) and use `terminal` for real consoles (when we show commands and their outputs).

**Comment by javiereguiluz** [comment]
Fantastic contribution, as always! Thanks Jérémy!

**Pull request merged**